projects/plugins/jetpack/modules/search/customberg/index.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
```

Comment:
Will we be able to rely on WordPress' `@wordpress/element` instead, just like we do when working with blocks?

Reply:
We should be able to, yes. I'll look into this in the follow-up PR.